packages/subgraph/src/utils.ts

```diff
@@ -11,7 +11,7 @@ export const BIG_INT_ONE = BigInt.fromI32(1);
 export const ZERO_ADDRESS = Address.zero();
 export let MAX_FLOW_RATE = BigInt.fromI32(2).pow(95).minus(BigInt.fromI32(1));
 export const ORDER_MULTIPLIER = BigInt.fromI32(10000);
-
+export const MAX_SAFE_MILLISECOND = BigInt.fromI64(8640000000000); //In seconds, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#the_ecmascript_epoch_and_timestamps
```

Comment:
maybe this should be called `MAX_SAFE_SECONDS` as the value is in seconds, then the comment can point out that the max safe milliseconds for the date object is: 8640000000000000 and include the link

Reply:
Done